<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositWindow</class>
 <widget class="QDialog" name="DepositWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>943</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DepositCalc</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>454</width>
        <height>917</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <layout class="QFormLayout" name="L1_add">
         <property name="labelAlignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="formAlignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>167</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Сумма вклада</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="l1_sumLayout">
           <property name="leftMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QLineEdit" name="lineSum">
             <property name="text">
              <string>100000</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>руб.</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>167</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Срок размещения</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="l2_periodLayout">
           <property name="leftMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QLineEdit" name="linePeriod">
             <property name="text">
              <string>12</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <property name="currentText">
              <string>месяцев</string>
             </property>
             <property name="currentIndex">
              <number>1</number>
             </property>
             <item>
              <property name="text">
               <string>дней</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>месяцев</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>лет</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>167</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Начало срока</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="l3_prdBeginLayout">
           <item>
            <widget class="QDateEdit" name="dateBox">
             <property name="dateTime">
              <datetime>
               <hour>12</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2022</year>
               <month>9</month>
               <day>5</day>
              </datetime>
             </property>
             <property name="currentSection">
              <enum>QDateTimeEdit::DaySection</enum>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
             <property name="date">
              <date>
               <year>2022</year>
               <month>9</month>
               <day>5</day>
              </date>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>167</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Процентная ставка</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="l4_intRateLayout">
           <property name="leftMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QLineEdit" name="lineInerestRate">
             <property name="text">
              <string>9</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>% годовых</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_7">
           <property name="minimumSize">
            <size>
             <width>167</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Налоговая ставка</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="leftMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QLineEdit" name="lineTaxRate">
             <property name="text">
              <string>13</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>%</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_8">
           <property name="minimumSize">
            <size>
             <width>167</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Переодичность выплат</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <layout class="QHBoxLayout" name="l5_pereodityLayout">
           <item>
            <widget class="QComboBox" name="comboBoxPeriodity">
             <property name="currentText">
              <string>раз в месяц</string>
             </property>
             <property name="currentIndex">
              <number>2</number>
             </property>
             <item>
              <property name="text">
               <string>каждый день</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>каждую неделю</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>раз в месяц</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>раз в квартал</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>раз в полгода</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>раз в год</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>в конце срока</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_11">
           <property name="minimumSize">
            <size>
             <width>167</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Капитализация</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <layout class="QHBoxLayout" name="l6_capitalizLayout">
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item>
            <widget class="QCheckBox" name="checkBoxCapital">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_10">
           <property name="minimumSize">
            <size>
             <width>167</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Список пополнений</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QPushButton" name="b_add_to_adds_list">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_6">
           <property name="minimumSize">
            <size>
             <width>167</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Ключевая ставка ЦБ</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <layout class="QHBoxLayout" name="l9_taxRateLayout">
           <property name="leftMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QLineEdit" name="lineCB">
             <property name="text">
              <string>8</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>%</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="L2_sub">
         <property name="labelAlignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="formAlignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_12">
           <property name="minimumSize">
            <size>
             <width>167</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Частичные снятия</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="b_add_to_subs_list">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="L3_bottom">
         <item row="1" column="0" colspan="2">
          <layout class="QHBoxLayout" name="l11_calcButLayout">
           <item alignment="Qt::AlignHCenter|Qt::AlignTop">
            <widget class="QPushButton" name="pushButtonCalc">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>Рассчитать</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0" colspan="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Начисленные проценты</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="l12_Layout">
           <item>
            <widget class="QLabel" name="label_out_percents">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Сумма налога</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="l13_Layout">
           <item>
            <widget class="QLabel" name="label_out_tax">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Сумма на вкладе к концу срока</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <layout class="QHBoxLayout" name="l14_Layout">
           <item>
            <widget class="QLabel" name="label_out_sum">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>График начисления процентов:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>300</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <column>
          <property name="text">
           <string>Дата</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Начислено
процентов,
руб.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Вклад
пополнен,
руб.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Остаток на
вкладе,
руб.</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineSum</tabstop>
  <tabstop>linePeriod</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>dateBox</tabstop>
  <tabstop>lineInerestRate</tabstop>
  <tabstop>lineCB</tabstop>
  <tabstop>pushButtonCalc</tabstop>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
